content/content/pages/vaccine-info.md

```diff
@@ -1,17 +1,23 @@
 Title: Where can I get the COVID vaccine in Chicago?
-Date: 03-19-2021
-Tags: vaccine
+Date: 04-03-2021
+Tags: vaccine, covid
 
-On March 29, Chicago will enter Covid vaccine [phase 1C](https://www.chicago.gov/city/en/sites/covid19-vaccine/home/vaccine-distribution-phases.html). This phase will include most Chicago residents, such as those working in retail, media, IT, communications, and real estate. There’s a good write-up on the current state of the vaccination effort [here](https://thetriibe.com/2021/03/most-chicagoans-will-be-eligible-for-vaccine-beginning-march-29-but-not-everyone-will-be-able-to-get-vaccinated-immediately/). All that said, many could likely get vaccinated before the mentioned date via one of the following methods:
+On March 29 Chicago entered Covid vaccine [phase 1C](https://www.chicago.gov/city/en/sites/covid19-vaccine/home/vaccine-distribution-phases.html) which made all essential workers and adults with underlying medical conditions eligible for COVID-19 vaccines. On April 12, vaccines will technically be made available to any [Illinois resident](https://coronavirus.illinois.gov/s/). This means that vaccines appointments may be difficult to get during April in Chicago.
+
+Given the high demand consider these methods for getting a Covid vaccine:
 
 - Call your primary care physician or main clinic, they may have extra vaccines or may have had appointment cancellations.
 - Join the [Chicago Vaccine Hunters Facebook Group](https://www.facebook.com/groups/1864372350383607/) — members post ways of getting appointments there regularly (including when particular websites get more appointments).
 - An [Illinois Vaccine Alert account on Twitter](https://twitter.com/ilvaccinealerts) has been created you can follow.
+- Check out the list of appointments in Cook County by a volunteer-run [site](https://www.ilvaccine.org/) for Illinois residents
+- Check out [Vaccine Spotter](https://www.vaccinespotter.org/IL/) which is [open source](https://github.com/getaaron/vaccine-spotter/)
+- Check out the [locations registered by the State of Illinois](https://coronavirus.illinois.gov/s/vaccination-location) that distribute vaccines
+- If you are unable to book an appointment on [Zocdoc](zocdoc.com/vaccine) for the [United Center](https://www.chicago.gov/city/en/sites/covid19-vaccine/home/united-center.html), call the United Center hotline (312) 746-4835 (This call center will be available from 8 a.m. to 8 p.m. Monday through Saturday and 8 a.m. to 4 p.m. on Sunday.)
```

Comment:
So, all of these newly added resources are already listed in the [Vaccine Angels spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ9rQdqgSJFapWJCKHA6ERzTva0W82DiEHKzG5AWkCNCaXwgwZOIgoIcN4gsdjgQr8doMLPRTfrLV_L/pubhtml?fbclid=IwAR1kxS9OC6uyiUo7H_ZAa13dXpP8rJ9rcxMdIHMKYljtl9BtS_deg3J4yjU#) in the below bullet point.

Do we think it's worth adding them separately above? We had previously removed all other specific links to appointment sites and had opted to instead reference the more thorough spreadsheet (which has a team of people keeping it up to date), but if you see a good reason to have the information in both places, we can update this.